Sources/Trap/Manager/Manager.swift

```diff
@@ -238,19 +243,23 @@ public class TrapManager {
     }
 
     @objc func batteryDidChange(_ notification: Notification) {
-        self.hasLowBattery =
-            (UIDevice.current.batteryState == .unplugged || UIDevice.current.batteryState == .unknown) &&
-            UIDevice.current.batteryLevel < config.lowBatteryThreshold &&
-            UIDevice.current.batteryLevel >= 0
+        setHasLowBattery()
         maybeModifyConfigAndRestartCollection()
 
         let batteryCollectorKey = String(reflecting: TrapBatteryCollector.self)
         guard let batteryCollector = (collectors[batteryCollectorKey] as? TrapBatteryCollector) else { return }
         batteryCollector.sendBatteryEvent()
     }
 
+    private func setHasLowBattery() {
+        self.hasLowBattery =
+            (UIDevice.current.batteryState == .unplugged || UIDevice.current.batteryState == .unknown) &&
+            UIDevice.current.batteryLevel < config.lowBatteryThreshold &&
+            UIDevice.current.batteryLevel >= 0    }
```

Comment:
Closing paren should go to the next line.

Reply:
This has been refactored